### PR TITLE
Remove Deploy class from the model

### PR DESCRIPTION
**Pull request by lukasz-golebiewski**
## Overview
Part of the cost accounting ticket. Parsed rholang code is no longer sent via grpc

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-2882

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by lukasz-golebiewski** [comment]
bors try

**Comment by lukasz-golebiewski** [comment]
bors try

**Comment by lukasz-golebiewski** [comment]
bors try

**Comment by lukasz-golebiewski** [comment]
bors retry

**Comment by lukasz-golebiewski** [comment]
bors try-

**Comment by lukasz-golebiewski** [comment]
bors try

File: casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala [hunk 1]
```diff
@@ -298,8 +299,8 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
       DeployData.toByteArray(ProtoUtil.stripDeployData(deploy))
     )
     // TODO: add error handling
-    val parsed = InterpreterUtil.mkTerm(deploy.term).right.get
     for {
+      parsed    <- Interpreter[F].buildNormalizedTerm(deploy.term)
```
Comment (ArturGajowy) [inline casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala]: :kissing_heart: :ok_hand:

File: casper/src/main/scala/coop/rchain/casper/PrettyPrinter.scala [hunk 2]
```diff
@@ -51,8 +51,8 @@ object PrettyPrinter {
   private def buildString(par: Option[Par]): String =
     par.map(p => limit(rpp.buildString(p), 25)).getOrElse("")
 
-  private def buildString(d: Deploy): String =
-    s"Deploy #${d.raw.fold(0L)(_.timestamp)} -- ${buildString(d.term)}"
+  private def buildString(d: DeployData): String =
+    s"DeployData #${d.timestamp}}"
```
Comment (ArturGajowy) [inline casper/src/main/scala/coop/rchain/casper/PrettyPrinter.scala]: What about the removed `${buildString(d.term)}"`? Maybe let's retain that, this seems like helpful debug info people are likely to rely on.

File: casper/src/main/scala/coop/rchain/casper/util/ProtoUtil.scala [hunk 3]
```diff
@@ -517,36 +507,19 @@ object ProtoUtil {
...
     )
 
+  def sourceDeployNow(source: String): DeployData =
+    sourceDeploy(
+      source,
+      System.currentTimeMillis(),
+      accounting.MAX_VALUE
+    )
+
   def compiledSourceDeploy(
```
Comment (ArturGajowy) [inline casper/src/main/scala/coop/rchain/casper/util/ProtoUtil.scala]: I think we should inline this to not cause the impression that the compilation process is avoided downstream.

File: casper/src/main/scala/coop/rchain/casper/PrettyPrinter.scala (hunk 2 above)
Comment (lukasz-golebiewski) [reply casper/src/main/scala/coop/rchain/casper/PrettyPrinter.scala]: Good catch. I was fixing compilation errors too hastily

**Pull request merged**